knora/dsplib/models/xmlproperty.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, node: etree.Element, valtype: str, default_ontology: Optional
                 # replace an empty namespace with the default ontology name
                 self._name = default_ontology + ':' + tmp_prop_name[1]
         else:
-            self._name = 'knora-admin:' + tmp_prop_name[0]
+            self._name = 'knora-api:' + tmp_prop_name[0]
```

Comment:
have you figured out why this used to be `knora-admin`? Did this ever happen? If so, something should be broken now, right?

Reply:
I assume it was a typo. Because knora-admin is obviously wrong. And due to this typo (among other factors), it was never possible to upload a Region/Annotation/LinkObj. But I didn't investigate it.